src/Pages/SubBreedPage.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from "axios";
+import { FunctionComponent, useEffect, useState } from "react";
+import ReactPaginate from "react-paginate";
+import { useParams } from "react-router-dom";
+import Dog from "../Component/Dog";
+//import * as helpers from "../Helpers";
+
+
+interface SubBreedPageProps {
+    
+}
+ 
+const SubBreedPage: FunctionComponent<SubBreedPageProps> = () => {
+    const POSTSPERPAGE = 10;
+    let {subBreed,breedName} = useParams<string>();
+    let [subBreedImgsContainer, setSubBreedImgsContainer] = useState<string[]>([]);
+    let [numOfPages,setNumOfPages] = useState<number>(0)
+    let [imgsInScreen,setImgsInScreen] = useState<string[]>([]);
+    const [compIsMounted,setCompIsMounted] = useState<boolean>(false); // Mounts once the fetchDogs function
+    const breedPageChange = ({selected}:any) => {
+        setImgsInScreen(subBreedImgsContainer.slice(selected*POSTSPERPAGE, selected*POSTSPERPAGE+POSTSPERPAGE))
+    }
+    useEffect( () => {
+        axios
+        .get('https://dog.ceo/api/breed/'+breedName+'/'+subBreed+'/images')

```

Comment:
How could you avoid the hardcoded writing of the api every time?

Reply:
Hi,
With a function, I have already created in the helpers but didn't utilize it in every place in my app. :)